### DISCUSSION TITLE
'모여락' 열기(=이벤트 열기) 네이밍을 코드에서 하나로 통일하는 게 어떨까요?

### DISCUSSION THREAD

**Discussion by compuTasha** [DeveloperAcademy-POSTECH/MacC-GetARock#45] [아이디어 제안하기]
<img width="142" alt="image" src="https://user-images.githubusercontent.com/51108729/202847792-ebcdb87f-5cfd-44ef-a7f1-8f1338dae917.png">

'모여락(=이벤트) 열기'가 현재 코드에서
gatherock, event 등 여러 이름으로 표현되고 있는데
헷갈리지 않도록 하나로 통일하는 게 어떨까요??

그리고 하나로 통일한다면 event가 가장 직관적인 것 같아서 한번 제안해봅니다...!

### 11월 20일 오전 9시 스크럼에서 'Gathering'으로 땅땅되었습니다~~🎉

**Comment by GODNOEL** [comment]
개발자 입장에서는 event이 더 직관적이여서 알아보기 편할 것 같긴해요!
저도 event에 동의합니다✌️

**Reply by skycat0212** [reply]
event가 오히려 개발자 입장에서 혼동을 줄 수 있을 것 같다는 생각도 들어요

**Reply by compuTasha** [reply]
앗 맞네요.. 개발에서의 event와 혼동될 것 같습니다.. 🥲

**Comment by EuniceNam** [comment]
프로그래밍적인 의미의 event와 혼동될까 걱정되는데, bandMeetup 어떤가요?

**Reply by skycat0212** [reply]
BandMeetUp은 '모임'이 저희 서비스의 가장 핵심적인 부분인데, 이를 나타내기에 너무 길게 풀어쓴 것 같다는 생각이 들어요.
게다가 MeetUp은 동사형태다보니 '모임'이라는 객체를 지칭하기에 어색한 느낌이 있는 것 같습니다.

**Reply by compuTasha** [reply]
Meetup이 설명회, 컨퍼런스, 간담회의 의미로 쓰이다 보니 조금 어색하게 느껴져요 🤔

**Reply by EuniceNam** [reply]
meetup은 사전에 "an informal meeting or gathering"으로 나오고, 비격식 만남으로 알고 있는데, 컨퍼런스로 쓰이나요..?

**Reply by compuTasha** [reply]
헉 너무 개발자 행사만 생각했네요..😳 그런 의미라면 밋업도 좋은 것 같아요!!

**Comment by skycat0212** [comment]
저도 프로그래밍적인 의미의 Event와 구분되도록 다른 표현을 사용해보는 것도 좋을 것 같아요.
그런 의미에서 Gathering 이 어떨까요?
<img width="259" alt="스크린샷 2022-11-19 오후 10 51 44" src="https://user-images.githubusercontent.com/20871603/202853978-405b301f-41e2-42d2-8a9e-376280b64907.png">
음악적 교류라는 특수한 목적을 위한 모임을 나타내는 명사형 표현이라는 측면에서 괜찮을 것 같아 제안드립니다.

**Reply by Somin-DS** [reply]
gathering좋은 것 같아요! 저희 겟어락이랑도 발음이 비슷하기도하고...이벤트와는 명확히 구분되기도 해서 저는 좋은 것 같습니다! band gathering이라고 검색해보니 밴드모임같은게 좀 나오는것같은데 bandGathering은 너무 길어질까요,,,? 그냥 게더링도 괜찮은 것 같습니다!

**Comment by compuTasha** [comment]
11월 20일 오전 9시 스크럼에서
'Gathering'으로 땅땅되었습니다~~
